dqc/system/sol.py

```diff
@@ -240,6 +243,32 @@ def requires_grid(self) -> bool:
     def getparamnames(self, methodname: str, prefix: str = "") -> List[str]:
         pass
 
+    def make_copy(self, **kwargs) -> Sol:
+        """
+        Returns a copy of the system identical to the orginal expect for new
+        parameters set in the kwargs.
+
+        Arguments
+        ---------
+        kwargs
```

Comment:
Same comment here about `**kwargs` instead of `kwargs`

Reply:
done